packages/react-components/src/atoms/__tests__/Link.test.tsx

```diff
@@ -199,3 +199,56 @@ describe('when button-styled', () => {
     expect(getByText('text').closest('a')).not.toHaveAttribute('href');
   });
 });
+
+describe('for elipsed links', () => {
+  // !! Unfortunately jest does not respect ellipsed texts (overflow: hidden)
+  // https://github.com/testing-library/react-testing-library/issues/751#issuecomment-691069646
+  // so we cannot test if the text is shown breaked as it should
+  // but we can test other properties and see it working in the StoryBook
+
+  describe('when child is a string', () => {
+    it('shows the title with the full text', () => {
+      const longText =
+        'A very long text showing that the elipsis feature is working well, trying to provide as much text here as I can to validate this feature.';
+      const { getByTitle } = render(
+        <div css={{ maxWidth: '10px' }}>
+          <Link href="/" ellipsed>
+            {longText}
+          </Link>
+        </div>,
+      );
+
+      expect(getByTitle(longText)).toBeInTheDocument();
```

Comment:
This is already tested. You should be using .closest('a').toHaveAttribute('title', 'expected string')